Game/Source/GDKTestGyms/Public/CounterComponent.h

```diff
@@ -14,6 +14,12 @@ class GDKTESTGYMS_API UCounterComponent : public UActorComponent
 {
 	GENERATED_BODY()
 
+	struct FActorCountInfo
+	{
+		int32 TotalCount;
+		int32 AuthCount;
```

Comment:
Capturing both auth and total actors here. I don't think we actually require the total count anymore.

If we want to log actor counts on clients, we might want to re add the log that was in UCounterComponent::UpdateCachedActorCounts. Then total count would be useful. 

Interested to hear peoples opinions.

Reply:
Seems useful to keep Total count around - valid for comparing against interest border size and subsequent performance impact